### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by masapr**
Added introductory text on the api-client

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # substrate-api-client
 
-substrate-api-client is a library written in Rust for connecting to the substrate's RPC interface via WebSockets allowing to
+The substrate-api-client is a Rust library for connecting to a [substrate](https://substrate.io/)-based node via RPC from a no-std environment. An RPC client is a library, with similar features as [Polkadot-js](https://wiki.polkadot.network/docs/polkadotjs). With an RPC client developers can easily interact with any [Polkadot](https://polkadot.network/) or [Kusama](https://kusama.network/) chain. There are [RPC clients](https://wiki.polkadot.network/docs/build-tools-index#rpc-and-api-tools) for different programming languages. For Rust, the most popular RPC client is [subxt](https://github.com/paritytech/subxt). The substrate-api-client uses core functionality of subxt and makes it available in a no-std environment.
```
Comment (haerdib) [inline README.md:3]: The substrate-api-client is a Rust library for connecting to a [substrate](https://substrate.io/)-based node via RPC. It provides similar functionalities as [Polkadot-js](https://wiki.polkadot.network/docs/polkadotjs), such as easy extrinsic submission and state queries. With an RPC client, developers can easily interact with any [Polkadot](https://polkadot.network/) or [Kusama](https://kusama.network/) chain. There are several [RPC clients](https://wiki.polkadot.network/docs/build-tools-index#rpc-and-api-tools) available in different programming languages. For Rust, the most popular RPC client is [subxt](https://github.com/paritytech/subxt). The substrate-api-client provides a simpler, less extensive alternative to subxt, focused on providing as many features as possible for no-std environments.

Comment (haerdib) [reply README.md:3]: Slightly changed wording - what do you think?

Comment (masapr) [reply README.md]: I added most of it. I still want to comment on "no-std" right at the beginning, so I added a sentence on that.

**Review by haerdib — APPROVED** [review]
Cool, thanks a lot!

**Pull request merged**